azure-pipelines-v3.yml

```diff
@@ -115,51 +115,26 @@ jobs:
       storage: opsbuildk8sprod
       containerName: docfx-bin
 
-# Integration test
-- job: IntegrationTest
+# Build integration test
+- job: BuildIntegrationTest
   dependsOn: WindowsBuild
   condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/v3'), eq(variables['System.PullRequest.TargetBranch'], 'v3')))
   pool:
     name: DocFX
```

Comment:
This can use a hosted windows vmImage. BTW, can you help change all the vmImage to latest ones like `windows-latest`?

Reply:
Though I think using hosted agent may take longer for cloning both repo.

Reply:
`--depth 1`, it gives us more parallelism.

Reply:
@yufeih This time doesn't seem acceptable :/
![image](https://user-images.githubusercontent.com/42199316/75212554-43bdb700-57c2-11ea-9ea9-8ab4effc55ec.png)

Reply:
More than 90s for dotnet build
![image](https://user-images.githubusercontent.com/42199316/75212617-7667af80-57c2-11ea-9677-6be6a8735c4d.png)